index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link href="style.css" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <header>
+    <a class="log">My Logo</a>
+    <nav class="ham">
+      <img src="img/Union.png" alt="bar">
+    </nav>
+  </header>
+  <main>
+    <section class="background-img row">
+        <div class="main">
+          <h1 class="main-description">
+            <br />I'm Tushar
+            <br />Glad to see
+            <br />you
+          </h1>
+        </div>
+        <p class="description">
+          I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+          experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <div class="text">let's connect</div>
+        <ul class="button-bott">
+          <li><a href=""><img class="twitter" src="img/twiter.png" alt="twiter_logo"></a></li>
+          <li><a href=""><img class="linkedIn" src="img/linkedin.png" alt="linkedIn_logo"></a></li>
+          <li><a href=""><img class="m" src="img/m.png" alt="m_logo"></a></li>
+          <li><a href=""><img class="github" src="img/github.png" alt="github_logo"></a></li>
+          <li><a href=""><img class="ok" src="img/ok.png" alt="ok_logo"></a></li>
+        </ul>
+    </section>
```

Comment:
[REQUIRED] Due to some changes you now have two spaces on the left and right.

Your Design

![image](https://user-images.githubusercontent.com/62938735/192116405-5bf180e4-aab6-4119-92a8-881091c1f488.png)